README.md

```diff
@@ -904,6 +904,18 @@ Thanks goes to these people ([emoji key][emojis]):
 This project follows the [all-contributors][all-contributors] specification.
 Contributions of any kind welcome!
 
+## Swag
```

Comment:
I prefer the contributors to be right above the license so it's easy to find (always at the bottom). Could you move this to between support and contributors? Otherwise this looks great!